app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/CompleteClaim.scala

```diff
@@ -935,7 +926,7 @@ object CompleteClaim {
         def isUKTax(taxCode: String): Boolean =
           TaxCode.listOfEUTaxCodes.map(t => t.toString()).exists(p => p.contains(taxCode))
         MoneyUtils.formatAmountOfMoneyWithPoundSign(
-          completeClaimsAnswer.claims.filter(p => isUKTax(p.taxCode)).map(s => s.claimAmount).sum
+          completeClaimsAnswer.filter(p => isUKTax(p.taxCode)).map(s => s.claimAmount).sum
```

Comment:
Lets rename `completeClaimsAnswer` to `claimsAnswer`

Reply:
Done